packages/ui/src/lib/table/Table.svelte

```diff
@@ -97,10 +97,12 @@ function sortImpl(): void {
 }
 
 onMount(async () => {
+  await tick(); // Ensure all initializations are complete.
```

Comment:
Maybe tick() should be moved immediately before sortImpl()? i.e. if no sorting then onMount() doesn't wait, if sorted then we mark the sort column immediately (no need for lines 106-110 in test) then tick and do the actual sort. Lower part of test would need to be rewritten.